src/style.css

```diff
@@ -0,0 +1,101 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  color: #3d4e61;
+  line-height: normal;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+form {
+  width: 100%;
+  max-width: calc(100% - 20px);
+  margin: 0 auto;
+}
+
+input[type="search"] {
+  width: 100%;
+  border: 1px solid #000;
+  border-radius: 5px;
+}
+
+[data-qa="big"] input[type="search"] {
+  width: 100%;
+  display: flex;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid hsl(210, 25%, 91%);
+  background: hsl(0, 0%, 100%);
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  overflow: hidden;
+}
+
+[data-qa="small"] input[type="search"] {
+  width: 100%;
+  height: 42px;
+  line-height: normal;
+  border-radius: 4px;
+  border: 1px solid hsl(210, 25%, 91%);
+  background: hsl(0, 0%, 100%);
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+input[type="search"]::placeholder {
+  background-repeat: no-repeat;
+}
+
+[data-qa="big"] input[type="search"]::placeholder {
+  font-size: 16px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+  line-height: normal;
+  background-image: url("./images/Search.svg");
+  background-size: 19px;
+  background-position: 26px center;
+  padding-left: 62px;
+}
+
+[data-qa="small"] input[type="search"]::placeholder {
+  font-size: 14px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+  background-image: url("./images/Search.svg");
```

Comment:
icons shouldn't be part of placeholder.
better to make it as pseudo element for label
actual
![chrome-capture-2023-6-29 (3)](https://github.com/mate-academy/layout_search-bar-airbnb/assets/33029864/dc7b3ead-56a1-4171-9841-469a3b14f9da)
expected
![expected](https://github.com/mate-academy/layout_search-bar-airbnb/assets/33029864/130663da-5d77-4131-837d-f44b1ec7c141)